### DISCUSSION TITLE
CPU Opt. Fan on ASUS ROG Crosshair VIII Dark Hero

### DISCUSSION THREAD

**Discussion by Pennijiro** [Rem0o/FanControl.Releases#1162] [Q&A] [answered]
Hi!

The CPU Optional Fan ASUS ROG Crosshair VIII Dark Hero (BIOS 4201) is recognized at Speeds (and RPM displayed) but not moved to Controls during assisted setup (also I can't find a manual way) - so effectively I can't control it (?). I have my case top fans plugged into it due to the vicinity of connector and the fans (saving cable routing effort) so I would of course like to control it based on the CPU temperature (to which they are also close). 

Is this behavior to expected? Something that might be added / fixed?

In any case, this really is a great piece of software! Thanks Rem0o!

Cheers,
Pennijiro

![Screenshot 2022-10-09 094912](https://user-images.githubusercontent.com/115393866/194744579-790798ba-c52e-492f-a87d-fac5fe16d021.jpg)

**Comment by Fr0stX76** [comment, marked as answer]
From your motherboard manual, CPU_OPT and CPU_FAN header share the same control. Not sure how this will work, but my guess is that CPU_FAN "control" will control both at the same time,.

So if you change the % of this control, it should adjust the FAN % (not RPM unless the FANs are identicals for both header) accordingly.

![Shared Controlpng](https://user-images.githubusercontent.com/70845235/194774433-98f7437e-35f0-42b7-9cd0-2bf8ba7f442b.jpg)

**Comment by Pennijiro** [comment]
Thanks for the response! Indeed, when I manually increase the CPU Fan, also the CPU Opt. Fan goes up accordingly. Sorry for not checking this myself and for bearing with me.
